akka-projection-core/src/main/scala/akka/projection/Projection.scala

```diff
@@ -11,7 +11,35 @@ import akka.annotation.ApiMayChange
 import scala.concurrent.{ ExecutionContext, Future }
 
 @ApiMayChange
-trait Projection {
+trait Projection[StreamElement] {
+
+  /**
+   * The projection group name. The name must be unique over the whole system.
+   * For instance, a 'user-view' could be the name of a projection.
+   */
+  def groupName: String
+
+  /**
+   * The unique key for this Projection instance. Must be unique within the projection group.
+   */
+  def key: String
```

Comment:
If you define a projection it will have a `name` or `groupName`. For instance, `user-view`. 

Then may have from 1 to many keys, for instance: `01`, `02` and `03`. 

So, at the end we will have three of those running. All with sharing the same name, but each consuming a subset of the events. `user-view-01` consumes the events in the bucket/partition 1, `user-view-02` those from bucket 2, etc.

Reply:
Got it.  I recall this from a zoom discussion now.  You can have multiple projections that have the same `groupName` or `name`, but you can only have one projection per `groupName` and `key`.